src/app.jsx

```diff
@@ -452,10 +453,12 @@ const NavigatorCardBody = ({
             }
         };
 
-        if (!isMounted.current) {
+        if (!isMounted.current && !Dialogs.isActive()) {
             isMounted.current = true;
             document.addEventListener("keydown", onKeyboardNav);
         }
+        if (Dialogs.isActive())
+            document.removeEventListener("keydown", onKeyboardNav);
```

Comment:
I suppose when the dialog closes, it should get added back. Where does this happen?

Reply:
When a dialog closes or opens the component is rerendered so the event listener gets added back